docs/using.md

```diff
@@ -468,7 +492,9 @@ spec:
 
 If the TaskSpec is provided, TaskRef is not allowed.
 
-See [the example TaskRun](../examples/runs/task-run.yaml).
+**Note**: TaskRun can embed both TaskSpec and resource spec at the same time. See [example](../examples/run/task-run-resource-spec.yaml) TaskRun.
```

Comment:
another way to structure this (not sure if it's better!) would be to have a section that clearly uses references, and another section that embeds both task spec and resource spec, and an example for each section

Reply:
@bobcatfish : let me know what you think about the docs update